From a32d88be3f404f8128e385327e187075bf334fde Mon Sep 17 00:00:00 2001
From: ariel aviad <aaviad@habana.ai>
Date: Sun, 22 Dec 2024 17:05:34 +0200
Subject: [PATCH] [FSW-6867] pald: remove reset hl target if eq pldm

Remove set HL_TARGET to palad if it equals pldm

Change-Id: I2986abf7ac8fe4264fc5cd43ae5e7c6526d1ebd7
Signed-off-by: ariel aviad <aaviad@habana.ai>
---

diff --git a/Makefile b/Makefile
index e7f4123..d68560f 100644
--- a/Makefile
+++ b/Makefile
@@ -12,11 +12,6 @@
 
 EMB_COMMON_MOD_DIR := ${PREBOOT_ROOT_DIR}/emb_common_mod
 
-# TODO: remove this after switching to palad target
-ifeq ($(HL_TARGET), pldm)
-    HL_TARGET = palad
-endif
-
 CFLAGS += -I$(SPECS_DIR)
 CFLAGS += -I$(SPECS_DIR)/$(BASE_DEV_NAME)/asic_reg
 CFLAGS += -I ${EMBEDDED_ROOT}/arcboot/arch
diff --git a/app/ext_rom/Makefile b/app/ext_rom/Makefile
index 2ccc555..ddd5abb 100644
--- a/app/ext_rom/Makefile
+++ b/app/ext_rom/Makefile
@@ -9,11 +9,6 @@
 
 EMB_COMMON_MOD_DIR := ${PREBOOT_ROOT_DIR}/emb_common_mod
 
-# TODO: remove this after switching to palad target
-ifeq ($(HL_TARGET), pldm)
-    HL_TARGET = palad
-endif
-
 CFLAGS += -I$(SPECS_DIR)
 CFLAGS += -I$(SPECS_EMBEDDED_DIR)
 CFLAGS += -I$(SPECS_DIR)/$(BASE_DEV_NAME)/asic_reg
diff --git a/app/palad_bloader/Makefile b/app/palad_bloader/Makefile
index 49bbf59..401b79b 100644
--- a/app/palad_bloader/Makefile
+++ b/app/palad_bloader/Makefile
@@ -10,11 +10,6 @@
 
 include $(HL_DEV_ARCH)/Makefile
 
-# TODO: remove this after switching to palad target
-ifeq ($(HL_TARGET), pldm)
-    HL_TARGET = palad
-endif
-
 CFLAGS += -I$(SPECS_DIR)
 CFLAGS += -I$(SPECS_DIR)/$(BASE_DEV_NAME)/asic_reg
 CFLAGS += -Wall -Werror
diff --git a/app/pcie_pif/Makefile b/app/pcie_pif/Makefile
index 1159854..450a792 100644
--- a/app/pcie_pif/Makefile
+++ b/app/pcie_pif/Makefile
@@ -8,11 +8,6 @@
 
 EMB_COMMON_MOD_DIR := ${PREBOOT_ROOT_DIR}/emb_common_mod
 
-# TODO: remove this after switching to palad target
-ifeq ($(HL_TARGET), pldm)
-    HL_TARGET = palad
-endif
-
 CFLAGS += -I$(SPECS_DIR)
 CFLAGS += -I$(SPECS_EMBEDDED_DIR)
 CFLAGS += -I$(SPECS_EMBEDDED_DIR)/$(BASE_DEV_NAME)
diff --git a/arch/gaudi2/common/Makefile b/arch/gaudi2/common/Makefile
index deb55db..5a9d97b 100644
--- a/arch/gaudi2/common/Makefile
+++ b/arch/gaudi2/common/Makefile
@@ -1,7 +1,3 @@
-# TODO: remove this after switching to palad target
-ifeq ($(HL_TARGET), pldm)
-    HL_TARGET = palad
-endif
 
 PREBOOT_SRCS_C += $(shell ls $(HL_DEV_ARCH_COMMON)/common.c)
 PREBOOT_SRCS_C += $(shell ls $(HL_DEV_ARCH_COMMON)/reset.c)
diff --git a/arch/gaudi3/arc/Makefile b/arch/gaudi3/arc/Makefile
index c1cab70..cb3bf8a 100644
--- a/arch/gaudi3/arc/Makefile
+++ b/arch/gaudi3/arc/Makefile
@@ -8,11 +8,6 @@
 
 CFLAGS += -I $(PREBOOT_ROOT_DIR)/lib/preboot
 
-# TODO: remove this after switching to palad target
-ifeq ($(HL_TARGET), pldm)
-    HL_TARGET = palad
-endif
-
 ifeq ($(HL_TARGET),onix)
 	LDFLAGS += -Wl,--just-symbols $(HL_DEV_ARCH)/exports_onix.elf
 else
diff --git a/arch/gaudi3/common/Makefile b/arch/gaudi3/common/Makefile
index a25c5e3..267877c 100644
--- a/arch/gaudi3/common/Makefile
+++ b/arch/gaudi3/common/Makefile
@@ -4,11 +4,6 @@
 CFLAGS += -I$(SPECS_EMBEDDED_DIR)/$(DEV_PREFIX)
 CFLAGS += -include $(SPECS_EMBEDDED_DIR)/$(DEV_PREFIX)/$(DEV_PREFIX)_reg_map.h
 
-# TODO: remove this after switching to palad target
-ifeq ($(HL_TARGET), pldm)
-    HL_TARGET = palad
-endif
-
 PPBOOT_SRCS_C += $(shell ls $(HL_DEV_ARCH_COMMON)/common.c)
 PPBOOT_SRCS_C += $(shell ls $(HL_DEV_ARCH_COMMON)/$(HL_TARGET).c)
 PPBOOT_SRCS_C += $(shell ls $(HL_DEV_ARCH_COMMON)/pll.c)
